### ISSUE TITLE
 Reveal in Explorer is not supported in the browser

### ISSUE THREAD

**Issue by nidrissi** [microsoft/vscode-remote-release#8568]
Type: <b>Bug</b>

The "Reveal in Explorer" command does not work when in a WSL workspace. Launching the command (directly or with the Shift+Alt+R shortcut) gives the following error message:

> Reveal in Explorer is not supported in the browser

Although I am working in the desktop version of VSCode.

Extension version: 0.79.2
VS Code version: Code - Insiders 1.79.0-insider (506cd5056d875ccdbea2e9a41ba7b9f19103d599, 2023-06-02T20:37:18.279Z)
OS version: Windows_NT x64 10.0.22631
Modes:
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz (4 x 3504)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.70GB (3.35GB free)|
|Process Argv|--crash-reporter-id 80bbdc64-3fe2-4f51-a379-d55f44c810f6|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: openSUSE-Tumbleweed|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-7600 CPU @ 3.50GHz (4 x 3503)|
|Memory (System)|7.61GB (6.28GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30404738
py29gd2263cf:30754470
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
pylantcb52:30590116
pyind779:30611226
pythonsymbol12:30651887
showlangstatbar:30737417
pythonms35:30671666
03d35959:30757351
24365598:30687740
pythonfmttext:30716741
pythoncmvfstr:30726892
fixhidewlkth:30724384
showsbindicator:30724477
pythongtdpath:30726887
pythonnosmt12:30757129

```

</details>

**Comment by diegovilar** [comment]
I'm not sure which version of the extension broke this feature, but it's probably a very recent one because I remember this still working somewhere last month (I use this feature quite often).

**Comment by JonrGull** [comment]
I swear I just used this a few days ago and it worked fine. Found out today that it doesn't work anymore and I get this error now.

**Comment by alexlee84** [comment]
I also have this problem today when I updated VS code to version 1.79.0. It worked well before.

**Comment by pybala** [comment]
I think we can still access from the explorer directly

`\\wsl.localhost\Ubuntu-22.04\opt\`

**Comment by KB3HNS** [comment]
Updated to Code 1.79.0 and WSL plugin v0.79.4 today.  I can confirm that this is now broken.  This has worked for years.

**Comment by safarli** [comment]
Same issue here, after the update 1.79.0

**Comment by ReveStobinson** [comment]
Came here to say the same thing as @safarli. Got `Reveal in Explorer` to start working again by downgrading the WSL extension to `v0.78.9` (or lower). 

It's possible the problem here may actually lie in the [WSL Extension](https://github.com/microsoft/vscode-remote-wsl-recommender). 

That repo doesn't have issues enabled since it's bundled with VSCode, so this one can probably stay where it is.

**Comment by ddh3** [comment]
![image](https://github.com/microsoft/vscode-remote-release/assets/30905704/ac4877ab-8fd4-4428-b417-2de87bd3ce82)

Confirmed, working on v0.78.9 for me as well, thank you all for the solution.

**Comment by yoratoni** [comment]
It seems to also cause issues with NextJS on WSL, hot reload is not working anymore, downgrading to `0.78.9` solves this problem too.

**Comment by jerryokk** [comment]
![image](https://github.com/microsoft/vscode-remote-release/assets/126859008/309e195c-36f6-4539-a2d3-bdd2f95dd5bf)

select v0.78.9

**Comment by AlexFUNBIT** [comment]
Confirmed, working on v0.78.9 for me too.

**Comment by Jkuffler** [comment]
Thanks for helping me figure this one out in no time. Rolled back WSL to 0.78.9 to continue using 'View in Explorer' in my Win11 File Explorer 👍 🔑

**Comment by chantlong** [comment]
Great! Rolling back to 0.78.9 did the trick! Thanks folks!

**Comment by Adesanya-Toba** [comment]
Same here. I'll roll back WSL for now but I'm definitely gonna be keeping an eye on this for a fix.

**Comment by aborruso** [comment]
Thank you, it was too bad that it didn't work, a bad bug.
Rolling back to 0.78.9 did the trick!

**Comment by aymalkhalid** [comment]
Rolled Back to 0.78.9 but for those who don't want to roll back can access WSL in the file explorer using:

> \\wsl$\Ubuntu-20.04\home


![image](https://github.com/microsoft/vscode-remote-release/assets/32593370/66530c98-7b7f-47b3-b0df-bdc938e6967f)

**Comment by connor4312** [comment]
WSL version 0.79.5 will be available shortly with the fix

**Comment by 377960738** [comment]
> WSL version 0.79.5 will be available shortly with the fix

Although this was fixed in 0.79.5, the response time of opening Explorer is still much slower than before (probably my illusion).
